tests/test_day_25_combo_breaker.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from icontract_hypothesis import test_with_inferred_strategy
+
+from aocdbc.day_25_combo_breaker import deduce_encryption_key
+from aocdbc.day_25_combo_breaker import transform
+
+
+class TestDay1(unittest.TestCase):
+    def test_part_1(self) -> None:
+        self.assertEqual(deduce_encryption_key(9093927, 11001876), 12227206)
+
+    def TODO_test_verify_with_icontract_hypothesis(self) -> None:
```

Comment:
Icontract-hypothesis probably often fails -- perhaps you can only test subfunctions, but not the main function?

Reply:
So `transform()` is already a smallish subfunction - I suspect icontract-hypothesis is doing a reasonable thing, but that hypothesis passes a large integer as `loop_size` and then the loop never completes.

Reply:
Maybe you can record this as well -- this would be one case where crosshair succeeds and icontract-hypothesis fails, right?

Reply:
It's sort of an odd situation - if there are bugs, hypothesis does find them (though I've seen the shrinking abort after 5 minutes).
The problematic case is when there aren't bugs - because hypothesis will continue to try values and easily picks one that is quite large and essentially hangs the process. So, all that is to say, I don't really have an example to put in recorded_failures for this. Unless you think I should copy the bug-free file in there or something?

Reply:
Can you just note it down somewhere for you, so that we can present it somewhere (a talk, a readme, crosshair or icontract-hypothesis documentation *etc.*)?

Reply:
Yup; left comment in test and keeping a log of notes for myself